util/dvsim/utils.py

```diff
@@ -600,7 +599,13 @@ def clean_odirs(odir, max_odirs, ts_format=TS_FORMAT):
     if os.path.exists(odir):
         # If output directory exists, back it up.
         ts = datetime.fromtimestamp(os.stat(odir).st_ctime).strftime(ts_format)
-        shutil.move(odir, "{}_{}".format(odir, ts))
```

Comment:
I am not sure this worked as intended before... i.e., the dir would be moved to a different hierarchy due to the missing parent path.

Reply:
It is used in quite a few places I believe. It works as intended. Lets meet to ensure the changes work as intended for all usecases?

Reply:
Sure we let's discuss this this week.

The reason why I think this may not have been working as intended is that the `if os.path.exists(odir):` is typically never evaluated when using time-stamps as the dir names, since they are all unique. When switchin this to `latest` this is not true anymore, and I noticed that dvsim was moving the old `local` dirs to the wrong hierarchy with the `shutil.move(odir, "{}_{}".format(odir, ts))` command, since `odir` contains the full path, but `ts` is only a name without the full path prepended to it.

Reply:
In the case of simulations, it's the `out` directory where the "latest" runs go, which would exist if the test was run previously. 

The `ts` is suffixed at the end, so appending it to a full path should still work.

Reply:
I think you are right and this should not be needed. 
Note however that all results folders except the newest one will then be called `latest_<ts>` instead of `<ts>`, since everytime we write a new result, we have to rename the old `latest` folder first.

Reply:
I've removed this change to minimize impact on already existing functions.